apps/web/lib/env.ts

```
@@ -445,6 +450,8 @@ const parsedEnv = createEnv({
     INVITE_DISABLED: process.env.INVITE_DISABLED,
     CHATWOOT_WEBSITE_TOKEN: process.env.CHATWOOT_WEBSITE_TOKEN,
     CHATWOOT_BASE_URL: process.env.CHATWOOT_BASE_URL,
+    FORMBRICKS_WORKSPACE_ID: process.env.FORMBRICKS_WORKSPACE_ID,
```

ESLint

[error] FORMBRICKS_WORKSPACE_ID is not listed as a dependency in turbo.json

(turbo/no-undeclared-env-vars)